substrate/frame/multisig/src/lib.rs

```diff
@@ -378,9 +405,15 @@ pub mod pallet {
 				maybe_timepoint,
 				CallOrHash::Call(*call),
 				max_weight,
+				None,
 			)
 		}
 
+		/// DEPRECATED: Use the `approve_as_multi` extrinsic instead.
```

Comment:
```suggestion
		/// DEPRECATED: Use the `approve_as_multi` extrinsic instead.
		/// It will be removed after March 2024.
```
I haven't looked at the entire PR but if this is moving forward we need to create a 'deprecation' issue just like this one https://github.com/paritytech/polkadot-sdk/issues/147 (following the process described here https://github.com/paritytech/polkadot-sdk/issues/182)

Reply:
From my understanding after reading the description of #182, shouldn't this be a note inside the `deprecated` attribute macro instead of a plain comment?

Reply:
True, that would help if a call to that function is compiled by Rust, which most likely would happen within a test.
Most of the cases the dispatchable function would be called by external Builders, in that case there isn't much we can do at the moment, please see https://github.com/paritytech/polkadot-sdk/issues/182#issuecomment-1691684159